### ISSUE TITLE
One person posted lie destroyed the reputation of a Colorado veteran 

### ISSUE THREAD

**Issue by Troyedad** [Troyedad/Troyeson#2]
This a  veteran's account about how one person's falsehood forced him to have a mental breakdown and separated from his little children, destroying his reputation and tarnishing his family name. I'm the veteran at the center of this story. As a naval hospital corpsman, I proudly served our country in the naval air reserve. My name is Troye Dwain Taylor, and I live in Aurora, Colorado. im giving you my experience because one person's posted untruth has had a significant detrimental impact on my children and my life. What hurts the most is that I served our nation and swore by the pledge I took to protect and defend our country from harm, but when I needed support from this country's inhabitants to remedy the injustice done to my family and I, I was forced to fight this battle alone. When I ask Colorado locals for assistance in redressing this injustice, they simply chuckle and drive away. Many Colorado residents find this injustice being done  to my family and me humorous. I became so angry with Colorado citizens that I regret ever joining the navy and serving my country after how I was treated by some of them. Now I understand how my father, a United States Marine, felt when he returned from his stint in Vietnam. Having a sense of emotion of feeling 
 used by your country. You believe that now that you have served your purpose, you no longer matter. I became so outraged over the situation, and so angry with some Colorado citizens, that I was admitted to the VA hospital for a psychological evaluation to assure that I was not a danger to others. People wonder why some soldiers suffer mental breakdowns as a result of foolish garbage like this. I'm not justifying their actions, but I do understand them now. Why do Colorado folks find it entertaining to laugh at someone who they can see iis in a poor situation and seeking assistance to remedy an injustice? It demonstrates an obvious lack of respect and gratitude for what I have had to endure on behalf of this country and its citizens. Very few
people will ever know what I have done and seen, so how can you judge me if you haven't walked in my shoes? What irritates me the most is when my fellow veterans simply turn their heads and ignore the situation. It was constantly hammered into me that no matter what, we never leave a fellow brother or sister behind; it is the idea we live by. This is not limited to being in combat. When you observe a fellow veteran in need, it is your obligation and moral duty to assist them. We veterans have each other's backs in battle and on the home front, which distinguishes us from everyone else. As aformer navy corpsman,  I would never abandon a fellow veteran. If I see a fellow veteran in need, it is my responsibility to assist that veteran in getting back on his or her feet, regardless of the circumstances. We stand together as one for all and one for all. Even though that should apply to my circumstance, the majority of Colorado citizens have already turned against me and refused to assist me in righting this wrong, so I'm left to fight this battle alone. Now I understand what it means when people say I love my nation; I only wish my country loved me as much as I love it. A few individuals have expressed gratitude for my service to our country, but in truth, actions speak louder than words. It's easy to say things, but acting on them is much more difficult. If you want to convey your gratitude, don't ignore the injustice done to my family and I . Stand up and tell everyone that what this person is doing and has done is completely wrong and unacceptable. It doesn't have to hurt for you to declare something is wrong and needs to be stopped. The truth is that if this injustice does not end, you could be the next victim. This is a story my grandmother told me, and it is relevant to the circumstance at hand. There are three men: one black man, one Chinese man, and one Jewish man. A man arrived one day to abduct the black man. Because I wasn't black, I said nothing. The next day, an arrived to pick up the Chinese man. I didn't say anything because I wasn't Chinese. On the third day, a man came to take me away without saying anything. A
wise person once stated that an injustice for one person is an injustice for everyone. The only thing necessary for the triumph of evil is for good men to do nothing.”  The
bottom line is that if this can happen to me, what makes you believe it won't happen to you? If you believe that, you are a fool.
